src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  width: 96px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
```

Comment:
use stars--5 modifier to bottom ( that everything was in order).
Also in this case you can make loop for better universality. ( optional)
Thanks)